conanfile.py

```diff
@@ -62,6 +64,7 @@ def _configure_cmake(self):
         cmake.definitions["HAVE_POSIX_FALLOCATE"] = True
         cmake.definitions["HAVE_STRERROR_R"] = True
         cmake.definitions["HAVE_USLEEP"] = True
+        cmake.definitions["DISABLE_MEMSTATUS"] = self.options.disable_memstatus
```

Comment:
Write it just after line `cmake.definitions["OMIT_LOAD_EXTENSION"] = self.options.omit_load_extension`